frontend/src/bundles/common/components/hint/hint.tsx

```diff
@@ -0,0 +1,47 @@
+import clsx from 'clsx';
+import { type FC } from 'react';
+import { Tooltip as ReactTooltip } from 'react-tooltip';
```

Comment:
Don't use components straight from library.
We already have Tooltip component for it

Reply:
But we can't pass any components in Tooltip as a property, only a text as a string, that's why i created a Hint component. So i still need to use Tooltip and modify it?